### ISSUE TITLE
Two Readiness checks

### ISSUE THREAD

**Issue by jahluwalia** [CloudNativeJS/cloud-health-connect#2]
I originally posted this in cloud-health, but have moved it here. 

I registered two readiness checks (one to check a valid mongodb connection, one to check a valid redis connection). If I call resolve on the second (redis), but not the first (mongo), my status is still UP. Shouldn't it be still STARTING? If i hit the /ready api, the returned checks array is emtpy.

If I swap out registerReadinessCheck for registerLIvenessCheck both the /ready and /health routes return both checks, but the states are still off. 

Here is the code:

I initialize as follows:
```js
const health = require('@cloudnative/health-connect');
    let healthcheck = new health.HealthChecker();
```

Create some readiness checks:

```js
//READINESS CHECK - MONGO
const dbConnection = new Promise(function(resolve, _reject) {
  var mongoose = require('mongoose');
  var mongoDB = 'mongodb://localhost:1234/db';
  mongoose.connect(mongoDB);
  mongoose.connect(mongoDB).then(
    () => {
      console.log('mongo connected');
      //resolve();
    },
    err => {
      //error
      console.log(err);
      _reject();
    }
  );
  // Get Mongoose to use the global promise library
  mongoose.Promise = global.Promise;

  //resolve();
});
let dbCheck = new health.ReadinessCheck('mongoCheck', dbConnection);
healthcheck.registerReadinessCheck(dbCheck);

//READINESS CHECK - REDIS
const redisConnection = new Promise(function(resolve, _reject) {

  var redis = require('./api/helpers/redisClient.js');

  redis.on('ready', function() {
    resolve();
    console.log('redis');
  });

  redis.on('error', function() {
    _reject();
  });

});
let redisCheck = new health.ReadinessCheck('redisCheck', redisConnection);
healthcheck.registerReadinessCheck(redisCheck);
```

And later on I do the following:

```js
app.use('/health', health.LivenessEndpoint(healthcheck));
app.use('/ready', health.ReadinessEndpoint(healthcheck));
```
 
In summary my questions are:
1) why is the status UP when the mongo check isn't calling resolve
2) why are the returned checks an empty array?
3) if i convert readinesscheck to livenesscheck, why do both health and ready routes show STARTING that contain both the mongo check and redis check?

Thanks,
Jas

**Comment by seabaylea** [comment]
@gdams can you take a look?

**Comment by gdams** [comment]
@jahluwalia thanks for the issue, I'm a little unsure about this issue. I'm not sure that commenting our `resolve();` will necessarily cause the health check to remain stuck at STARTING.

**Comment by gdams** [comment]
fixed in `1.0.2`
